cmd/run.go

```diff
@@ -301,6 +305,14 @@ func VerifyConfig(cfg *Config) error {
 		return fmt.Errorf("config 'http.upstreamTimeout' (%s) cannot be lower than 'listObjectsDeadline' config (%s)", cfg.HTTP.UpstreamTimeout, cfg.ListObjectsDeadline)
 	}
 
+	if cfg.Log.Format != "text" && cfg.Log.Format != "json" {
+		return fmt.Errorf("log format must be either 'text' or 'json'")
```

Comment:
config 'log.format' must be one of ['text' or 'json']

Reply:
Updated. Thanks!